packages/cli/test/utils/simulation/SimulationTracker.ts

```diff
@@ -1,36 +1,202 @@
 import EventEmitter from "node:events";
 import {routes} from "@lodestar/api/beacon";
-import {altair, Epoch, Slot} from "@lodestar/types";
+import {TIMELY_HEAD_FLAG_INDEX, TIMELY_TARGET_FLAG_INDEX, TIMELY_SOURCE_FLAG_INDEX, ForkName} from "@lodestar/params";
+import {allForks, altair, Epoch, Slot} from "@lodestar/types";
 import {toHexString} from "@lodestar/utils";
 import {EpochClock} from "./EpochClock.js";
 import {BeaconNodeProcess, SimulationParams} from "./types.js";
-import {
-  avg,
-  computeAttestation,
-  computeAttestationParticipation,
-  computeInclusionDelay,
-  computeSyncCommitteeParticipation,
-  getForkName,
-} from "./utils.js";
-
-const participationHeading = (id: string): string => `${id}-P-H/S/T`;
-const missedBlocksHeading = (id: string): string => `${id}-M`;
-// const nodeHeadHeading = (id: string): string => `${id}-H`;
-const finalizedHeading = (id: string): string => `${id}-F`;
-const syncCommitteeHeading = (id: string): string => `${id}-SC`;
+import {avg, getForkName} from "./utils.js";
+
+const TIMELY_HEAD = 1 << TIMELY_HEAD_FLAG_INDEX;
+const TIMELY_SOURCE = 1 << TIMELY_SOURCE_FLAG_INDEX;
+const TIMELY_TARGET = 1 << TIMELY_TARGET_FLAG_INDEX;
+
+type SlotMeasureInput = {
+  version: ForkName;
+  slot: Slot;
+  node: BeaconNodeProcess;
```

Comment:
can we keep `node` out of this type, and pass node as the separate first argument whereever SlotMeasureInput is being passed currently